src/main/resources/pl/damianszczepanik/jenkins/buildhistorymanager/model/Rule/help-continueAfterMatch.html

```diff
@@ -1,22 +1,25 @@
 <p><b>Description</b></p>
 <p>
-    By default, after the conditions of a rule are met and the action are apllied,
-    the plugin moves on to the next completed build in the build history and
-    starts to process all the rules again from the start of the list of rules.
-</p>
-<p>
-    When this flag is set to <code>false</code> (default) AND this rule's conditions are met,
-    this rule is the last rule applied to the build. The plugin moves on to the next completed build
-    and starts processing the rules from the start of the list of rules.
-</p>
-<p>
-    When this flag is set to <code>true</code> AND this rule's conditions are met,
-    the plugin moves on to the next rule with the same build. This way multiple consecutive rules
-    can be applied to the same build.
+    By default, once the conditions of a rule are met and the corresponding actions are applied,
+    the plugin proceeds to the next completed build in the build history and
+    triggers a reevaluation of all rules, starting from the beginning of the rules list.
 </p>
+<ul>
+    <li>
+        When this flag is set to <code>false</code> (which is the default),
```

Comment:
Maybe this is not clear to me but this sentence is in opposite to above - in case what is default value for this flag

Reply:
The paragraph above as being wrong too still. "By default, once ...". It says the opposite as what the rest says.

Reply:
The first paragraph` "By default, once the conditions ..."` was revised to align with the original description. However, it appears that this description is not accurate for the `continueAfterMatch (default true)` . I am considering deleting the first paragraph from the help file. 
@damianszczepanik and @martinda  what are your thoughts?

Reply:
I am okay with taking the first paragraph out. In a future pull request, the pipeline nomenclature can still be added to further clarify and help users.